### PR TITLE
App setup script, and a small thing about deactivating plugin when missing dependency

### PR DESCRIPTION
**Pull request by joshkadis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ When the post is rendered on the front end, this same data is used to bootstrap
 ### Installation on WordPress.com VIP
 
 1. Place the plugin in a directory inside your theme, e.g.  `mytheme/inc/wordpress-simplechart`
-1. Follow the **Local app setup commands** steps below
+1. Follow the **Local app setup script** steps below
```
Comment (joshkadis) [inline README.md]: @dmachat Having done this now, I'm thinking that the default should be to ship the plugin _with_ the `/app` directory already installed from `/client/pages` in the Simplechart repo, then we'd only use this script when we needed to update it.

Comment (dmachat) [inline README.md]: yeah, that makes sense. we'd only need to include and version the compiled js and index.html

Comment (joshkadis) [inline README.md]: right, this webpack noob keeps forgetting that main.bundle.js has all the stylesheets and images and everything

**Comment by dmachat** [comment]
:ice_cream:

**Comment by joshkadis** [comment]
@dmachat I didn't commit and push the `simplechart/client/pages` directory yet to keep the code review manageable, let me know what you think of this, then i can add that stuff

**Comment by dmachat** [comment]
:trophy:

**Pull request merged**